src/ppx/ppx_eliom_utils.ml

```diff
@@ -879,7 +906,7 @@ module Make (Pass : Pass) = struct
         let l = flatmap (dispatch_str c mapper) strs in
         maybe_reset_injected_idents c ; l
       | Pstr_extension (({txt}, PStr strs), _) 
-        when is_annotation txt ["rpc"; "crpc" ;"crpc_opt"] ->
```

Comment:
The eliom rpc is supposed to be the default. I would specialize all ocsigenstart extension naming and name eliom one's "rpc".

"cw_rpc" might a good pick for connected wrapper.